csrc/scheduler/normalization_inner_outer.cpp

```diff
@@ -147,17 +151,17 @@ std::unique_ptr<ReductionParams> getInnerOuterPersistentHeuristics(
         runtime_info,
         data_cache,
         reduction_tvs,
-        hp.vectorize_factor,
-        hp.threads_per_block_min,
-        hp.threads_per_block_max,
-        warp_specialized);
+        vectorize_factor,
+        threads_per_block_min,
+        threads_per_block_max,
+        user_enforced_warp_specialization);
```

Comment:
Why is this change? This lambda gets `warp_specialized` as its parameter but doesn't seem to use it.

Reply:
nice cache. Shouldn't change this. I inlined this lambda for clarity.